courses/templates/course_list_to_add.html

```diff
@@ -12,12 +12,14 @@
     <th>Add</th>
   </tr>
   {% for c in courses %}
+  {% if c.student_registration_open %}
```

Comment:
don't put this condition here,
put the condition in the query set filter
`Course.objects.filter(student_registration_open=True)`